src/sentry/integrations/slack/utils/escape.py

```diff
@@ -0,0 +1,17 @@
+# TODO(scttcper): Might need to add "*" bold, and "_" italics
+ESCAPE_CHARS = [
+    # Replace ampersand first to avoid adding more of them
+    ("&", "&amp;"),
+    ("<", "&lt;"),
+    (">", "&gt;"),
+]
```

Comment:
Is this specific to slack? Or should we just urlencode the entire string?

Reply:
i think it will continue to be specific to slack and i don't think i want url encode, this is more like html encode

Reply:
we might have to do something with * and _ characters? not sure yet. bold and italics

Reply:
Ok, so I didn't realise `mrkdown` was a slack specific language, I thought it was a typo haha. 

From their docs:
> You shouldn't HTML entity-encode the entire text, as only the specific characters shown above will be decoded for display in Slack.

So what you've done seems fine, might be worth linking to their docs